commit 6f3a20c731dbde18a433a1a1502c006b87a6acb4 (refs/changes/95/4495/2)
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2018-08-24T14:23:46+00:00 (1 year, 1 month ago)
    
    joyent/node-mooremachine#17 package.json repository does not point to Joyent
    Reviewed by: Alex Wilson <alex.wilson@joyent.com>
    Approved by: Alex Wilson <alex.wilson@joyent.com>

diff --git a/package.json b/package.json
index a3b1ac2..379f99a 100644
--- a/package.json
+++ b/package.json
@@ -8,7 +8,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/arekinath/node-mooremachine.git"
+    "url": "git+https://github.com/joyent/node-mooremachine.git"
   },
   "author": "Joyent, Inc",
   "contributors": [
